tests/test_trec_indexer.py

```diff
@@ -26,6 +26,7 @@ def tearDown(self):
 
     def test_TREC_indexing(self):
         indexer = pt.TRECCollectionIndexer(self.test_dir)
+        pt.redirect_stdouterr()
```

Comment:
Was this intended? It should be default now

Reply:
yep. debugging leftover.